src/main/java/pw/kaboom/extras/modules/server/ServerTick.java

```diff
@@ -1,19 +0,0 @@
-package pw.kaboom.extras.modules.server;
-
-import org.bukkit.World;
-import org.bukkit.event.EventHandler;
-import org.bukkit.event.Listener;
-import org.bukkit.event.player.PlayerStatisticIncrementEvent;
-
-public final class ServerTick implements Listener {
-    @EventHandler
-    void onPlayerStatisticIncrement(final PlayerStatisticIncrementEvent event) {
-        final World world = event.getPlayer().getWorld();
-
-        if (!world.isAutoSave()) {
-            world.setAutoSave(true);
-        }
```

Comment:
I didn't think turning on the autosave was necessary, since the world is pretty volatile on Kaboom (playerdata isn't kept, and the world is frequently reset). I can place it back if you want, though.

Reply:
I don't think the auto-save functionality actually removes all the block changes from memory, but I re-introduced the auto-save enabler just in case. Now it does it every 30 seconds instead of every tick.